code/__DEFINES/traits.dm

```diff
@@ -628,6 +630,8 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 #define MAGIC_TRAIT "magic"
 #define TRAUMA_TRAIT "trauma"
 #define FLIGHTPOTION_TRAIT "flightpotion"
+#define EMOTE_TRAIT "emote"
+#define DRUNK_TRAIT "drunk"
```

Comment:
I guess I won't require it but in the future I always prefer to use `[type]` for this one off stuff.

Reply:
Thanks for the review! I'm not too familiar with this, but I've replaced the trait sources with "[type]" and it looks to be functioning just as before. Hopefully that lines up with how it should be used.